timbot.py

```diff
@@ -81,11 +81,20 @@ def run_timbot():
 					# what time is lunch
 					elif 'lunch' in message and ('time' in message or 'when' in message):
 						sc.api_call("chat.postMessage", channel=channel_name, text=ideal_lunch_time, as_user=True)
-						uploadimage('images/lunchchart.png', 'IdealLunchTimeChart')
+						uploadimage('images/lunchchart.png', 'IdealLunchTimeChart','')
 
 					# what to eat
 					elif 'what' in message and 'eat' in message:
-						sc.api_call("chat.postMessage", channel=channel_name, text="chicken sandwich", as_user=True)
+						# if friday
+						if datetime.datetime.today().weekday() == friday_index_elem:
+							sc.api_call("chat.postMessage", channel=channel_name, text="Its friday enjoy a meal out. Maybe some french toast at pauls?", as_user=True)
+						else:
+							image_url = 'http://cafe.epicureanfeast.com/Clients/8680redhat.jpg'
+							attachments = [{"title": 'Menu', "image_url": image_url}]
+							responce = sc.api_call("chat.postMessage", channel=channel_name, text='Heres the cafe menu', as_user=True, attachments=attachments)
```

Comment:
You don't need to assign to this variable.

Reply:
fixed